fixtures/room/query_room_fixtures.py

```diff
@@ -15,9 +15,37 @@
     "data": {
         "allRemoteRooms": {
             "rooms": []
-            }
         }
     }
+}
+
+all_remote_rooms_query_with_returnAll_True = '''
+ query{
+  allRemoteRooms(returnAll: true){
+    rooms{
+      calendarId
+      name
+    }
+  }
+}
+'''
+
+all_remote_rooms_response_with_returnAll_True = {
```

Comment:
Please look into your variable naming. Read the conventions -> https://www.python.org/dev/peps/pep-0008/#function-and-variable-names